src/ensembl/src/Root.tsx

```diff
@@ -53,7 +53,7 @@ const Root: FunctionComponent<RootProps> = (props: RootProps) => {
   }, [cookies]);
 
   return (
-    <BrowserRouter>
+    <React.Fragment>
```

Comment:
Two questions:
- why do we need Fragment here (there is only one child, not an array of children)
- should we write this as `React.Fragment` or using the shorthand `<> ... </>` notation? I am a bit partial to the shorthand notation (once I tried it, I never wanted to go back), but would like to know what you think. @imran-ebi ?

Reply:
That's probably because I was too lazy to remove it, after the `div` tag was added. We can definitely remove it. As for fragments, I prefer the `Fragment` without the `React`.

I don't have an opinion about the shorthand, since I've never used it. Won't it look hacky compared to using a name tag?

Reply:
> Won't it look hacky compared to using a name tag?
I don't think it will — it's a standard React tag after all. I used Fragment without React for a long time too; then I tried the shorthand, and now I'm a strong believer in it.

Reply:
I too would prefer <Fragment> as it will be a little more readable.

Reply:
Not that I want to push either of you one way or the other; I just want to offer myself as an example of someone who, conservatively, thought that `Fragment` is more readable and that the empty angle brackets look a bit weird and unsatisfying — and then after the majority of our team decided on the shorthand, tried it, and found it very liberating. No more expressly declared imports from `react` or using the `React` namespace; it was like having to import something from `react` to write a `<div>` in JSX, and then being relieved from that effort.

It took about 10 minutes to get used to empty angle brackets :smile: